template-js/template/index.js

```diff
@@ -1,5 +1,5 @@
-import { AppRegistry } from 'react-native'
 import 'react-native-gesture-handler'
```

Comment:
why we need to import the whole package here?

Reply:
It's required by the library

Reply:
https://reactnavigation.org/docs/getting-started#installing-dependencies-into-a-bare-react-native-project